tests/arch/arm64/arm64_high_addresses/CMakeLists.txt

```
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.1)
```

We have 3.20.0 for the rest of the tests and samples.
```suggestion

cmake_minimum_required(VERSION 3.20.0)
```